servers/rendering/renderer_rd/shaders/environment/volumetric_fog.glsl

```diff
@@ -232,7 +232,7 @@ void main() {
 
 #ifdef DENSITY_USED
 		density *= cull_mask;
-		if (abs(density) > 0.001) {
+		if (abs(density) > 0.00005) {
```

Comment:
Anything below ``0.000976562`` (1 / DENSITY_SCALE) will be rounded down to 0, so there is really no point in running this code if density is less than ``0.001``

Reply:
This explains why FogVolume's density property has no effect below `0.001` :slightly_frowning_face:

Reply:
Does this apply to FogVolumes set to "World (Global)" as well? I'm trying to use a shader for global fog, and anything below `0.001` isn't rendering.